docs/guide/results/working_with_networks.rst

```diff
@@ -3,8 +3,56 @@
 Working with networks of results
 ================================
 
-issue #803
+After running a **network** of free energy calculations,
+we often want to consider analysing the corresponding network of results.
 
+.. _userguide_MLE:
 
-The :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
+Converting relative results to absolute estimates
+-------------------------------------------------
+
+When a network of relative free energies have been calculated,
+a commonly performed task is to transform these pairwise estimations of **relative** free energies
+into a **absolute** free energies.
+This is done using a maximum likelihood estimator (MLE) [1]_,
+as implemented in the `cinnabar`_ package.
+This approach uses the matrix of relative pairwise measurements and their uncertainties,
+to estimate the overall ranking of
+To use this approach the network of pairwise measurements needs to be fully connected,
+i.e. there should be a way to trace a path along pairwise measurements between any two nodes (ligands) on the network.
+
+.. note::
+   The results of a MLE estimation will have a **mean** of 0.0,
+   meaning that there will be some estimates with positive values and some estimates with negative values.
+   These predictions (:math:`\Delta G_{pred}`) can be shifted to match the magnitude of the experimental data,
+   to satisfy the below equation where the sum is performed over N molecules that have experimental data (:math:`\Delta G_{exp}`) [2]_.
+
+   .. math::
+
+      \sum_i^N \Delta G^i_{exp} = \sum_i^N \Delta G^i_{pred}
+
+Gathering using the command line
+--------------------------------
+
+After running calculations using the :ref:`quickrun command <userguide_quickrun>`,
+the :ref:`openfe gather <cli_gather>` command offers a way to collate information across many different individual
 simulations and prepare a table of results.
+The tool offers a summary of the relative binding affinities,
```

Comment:
Would it make sense to add the command line option here?, e.g.
```suggestion
The tool offers a summary of the relative binding affinities (`--report ddg`),
```
and then in the next line ``--report dg``